core/test-api/src/main/kotlin/testApi/testRunner/TestRunner.kt

```diff
@@ -85,7 +85,7 @@ abstract class AbstractTest<M : TestMethods, T : TestBuilder<M>, D : DokkaTestGe
                     }.toSet(),
                     sourceLinks = sourceSet.sourceLinks.map { link ->
                         link.copy(
-                            localDirectory = testDirPath.toFile().resolve(link.localDirectory).canonicalPath
+                            localDirectory = testDirPath.toFile().resolve(link.localDirectory).absolutePath
```

Comment:
The problem with paths was here. `canonicalPath` resolves symlinks, while `absolutePath` does not, so on windows in resulted in two different temp folder locations.

I'll refactor this bit later on in a separate PR once I get a hold of a windows machine